styles.css

```diff
@@ -0,0 +1,595 @@
+  text-decoration: none;
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+.author a {
+  color: #000;
+  text-decoration: underline;
+  text-decoration-style: dotted;
+  font-weight: bold;
+}
+
+.right-div a {
+  font-family: Georgia, "Times New Roman", Times, serif;
+  font-size: 17px;
+  line-height: 23px;
+  color: #000;
+  text-decoration: none;
+}
+
+.left a:visited {
+  color: black;
+}
+
+.right button {
+  margin: 0 5px;
+  background-color: #567b95;
+  color: #fff;
+  padding: 11px 12px 8px 12px;
+  border: none;
+  border-radius: 3px;
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 11px;
+}
+
+/* -------------------- Main Section --------------- */
+
+.article-title {
+  width: 600px;
+  margin: 50px auto 15px auto;
+}
+
+.main-image {
+  max-width: 1200px;
+  width: 100%;
+  margin: auto;
+}
+
+.main-image img {
+  width: 100%;
+}
+
+figcaption {
+  font-size: 15px;
+  line-height: 20px;
+  margin-right: 7px;
+  color: #666;
+  font-family: georgia, "times new roman", times, serif;
+}
+
+figcaption span {
+  font-size: 13px;
+  color: #888;
+  letter-spacing: 0.13px;
+}
+
+.container {
+  max-width: 100%;
+  margin-top: 50px;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  margin-left: calc((100% - 600px) / 2);
+}
+
+.container > div {
+  max-width: 600px;
+}
+
+.lower-footer {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px 0;
+}
+
+.author {
+  color: #000;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.author a:hover {
+  text-decoration: none;
+}
+
+.social-media-icons i {
+  margin: 0 4px;
+  background-color: #eee;
+  border-radius: 50%;
+  padding: 5px;
+  border: 1px solid #eee;
+}
+
+.social-media-icons .fa-facebook-f {
+  width: 26px;
+  height: 26px;
+  padding: 4px 0 0 6px;
+}
+
+.social-media-icons i:hover {
+  background-color: transparent;
+  cursor: pointer;
+}
+
+.social-media-icons .far {
+  color: rgb(102, 102, 102);
+  font-size: 22px;
+  font-weight: 50;
+  background-color: transparent;
+  border: none;
+}
+
+.social-media-icons .far:hover {
+  color: #eee;
+  cursor: pointer;
+}
+
+.main-text {
+  font-size: 20px;
+  line-height: 30px;
+  color: #333;
+  margin-bottom: 20px;
+}
+
+.separation {
+  width: 72px;
+  border-bottom: 1px solid rgb(204, 204, 204);
+  margin: 0 auto 18px auto;
+}
+
+ul {
+  list-style: none;
+  font-size: 15px;
+  line-height: 19px;
+  text-align: center;
+  font-weight: bold;
+}
+
+.subscribe {
+  display: block;
+  width: 169px;
+  margin: 3px auto 18px auto;
+  color: #326891;
+  font-size: 15px;
+  line-height: 19px;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+/* -------------------- Aside Section --------------- */
+
+aside {
+  width: 210px;
+  margin-left: 30px;
+}
+
+.aside-articles {
+  width: 100%;
+  position: relative;
+  top: 1000px;
+  z-index: 10;
+}
+
+.aside-title {
+  border-bottom: 1px solid #eee;
+  padding-bottom: 10px;
+  margin-bottom: 10px;
+}
+
+.picks article {
+  width: 100%;
```

Comment:
- [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)